bezier-rs/docs/interactive-docs/src/App.vue

```diff
@@ -250,6 +237,34 @@ export default defineComponent({
 						});
 					},
 				},
+				{
+					name: "Hull",
+					callback: (canvas: HTMLCanvasElement, bezier: WasmBezierInstance, options: Record<string, number>): void => {
+						const hullPoints: Point[][] = JSON.parse(bezier.hull(options.t));
+						hullPoints.forEach((iteration: Point[]) => {
+							iteration.forEach((point: Point, index) => {
+								drawPoint(getContextFromCanvas(canvas), point, 3, COLORS.NON_INTERACTIVE.STROKE_1);
+								if (index === 0) {
+									return;
+								}
+								const prevPoint: Point = iteration[index - 1];
+								drawLine(getContextFromCanvas(canvas), point, prevPoint, COLORS.NON_INTERACTIVE.STROKE_1);
+							});
+						});
+					},
+					template: markRaw(SliderExample),
+					templateOptions: {
+						sliders: [
+							{
+								min: 0,
+								max: 1,
+								step: 0.01,
+								default: 0.5,
+								variable: "t",
```

Comment:
I think @hannahli2010's comment was to use `templateOptions: { sliders: [tSliderOptions] },` here instead.

Reply:
OH, I misunderstood thanks for pointing that out